arPullGlob.py

```diff
@@ -192,7 +159,9 @@ def cmFlts():
     stDHi=[]
 
     aaaa=list(resultsIntlk.keys())
```

Comment:
a general TODO re: variable names hahahaha

Reply:
Indeed.  I just wanted to get it working.